src/controllers/EventsControllerV2.js

```diff
@@ -0,0 +1,159 @@
+// @flow
+
+import type { Event, EventData } from '../types';
+import type EventManager from '../managers/EventManager';
+
+import Controller from './Controller';
+import anonymous from '../decorators/anonymous';
+import route from '../decorators/route';
+import httpVerb from '../decorators/httpVerb';
+import serverSentEvents from '../decorators/serverSentEvents';
+import eventToApi from '../lib/eventToApi';
+import Logger from '../lib/logger';
+const logger = Logger.createModuleLogger(module);
+
+const KEEP_ALIVE_INTERVAL = 9000;
+
+class EventsControllerV2 extends Controller {
+  _eventManager: EventManager;
+  _keepAliveIntervalID: ?string = null;
+  _lastEventDate: Date = new Date();
+
+  constructor(eventManager: EventManager) {
+    super();
+
+    this._eventManager = eventManager;
+  }
+
+  @httpVerb('post')
+  @route('/v2/ping')
+  @anonymous()
+  async ping(payload: Object): Promise<*> {
```

Comment:
Is this endpoint required? I don't think I added a call to it.

Reply:
nope, I copy pasted, I don't use it anywhere.